pgp/openpgp.go

```diff
@@ -94,12 +92,6 @@ func checkDetachedSignature(keyring openpgp.KeyRing, signed, signature io.Reader
 			sigType = sig.SigType
 			creationTime = sig.CreationTime
 			pubKeyAlgo = sig.PubKeyAlgo
-		case *packet.SignatureV3:
```

Comment:
Will this have any impact?

Reply:
Yes something I should have made clearer in the PR description. Basically this means we would lose support for SignatureV3. They are old and nearly unsupported so I think it's your decision.

On the other hand, the current openpgp does not support ed25519 keys ([example issue](https://github.com/gopasspw/gopass/issues/2192)), and is not supported. So migrating has both upsides and downsides.

I searched for another maintained `/x/crypto/openpgp` alternatives that is still maintained without success sadly

Reply:
After double checking https://github.com/ProtonMail/go-crypto/blob/afb1ddc0824ce0052d72ac0d6917f144a1207424/openpgp/packet/packet.go#L339, I think there was already now way to read a SignatureV3 from a packet, so I think we don't lose anything.

Here is a public issue showing a similar intel https://github.com/golang/go/issues/27679

Reply:
@neolynx any  input on this, otherwise I think we can merge

Reply:
I would go with that. We can test with old mirrors and repos, and readd it in case this brings any troubles...